manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
There is a space between `python` and the shebang path. It should be `#!/usr/bin/env python` without the space to ensure the script runs correctly.